libraries/abstractions/secure_sockets/cellular/iot_secure_sockets.c

```diff
@@ -1128,12 +1230,32 @@ int32_t SOCKETS_Connect( Socket_t xSocket,
     {
         serverAddress.ipAddress.ipAddressType = CELLULAR_IP_ADDRESS_V4;
 
-        /* This macro is defined in secure sockets header file.
-         * It should be used during address convert in secure sockets implementation. */
-        /* coverity[misra_c_2012_directive_4_6_violation] */
-        /* coverity[misra_c_2012_rule_21_6_violation] */
-        ( void ) SOCKETS_inet_ntoa( pxAddress->ulAddress, serverAddress.ipAddress.ipAddress );
-        serverAddress.port = pxAddress->usPort;
+        #if ( CELLULAR_SUPPORT_GETHOSTBYNAME == 0 )
+            pHostname = prvReverseLookup( pxAddress->ulAddress );
+
+            if( pHostname != NULL )
+            {
+                strncpy( serverAddress.ipAddress.ipAddress, pHostname, CELLULAR_IP_ADDRESS_MAX_SIZE );
```

Comment:
Same as above.

Reply:
This will be updated in a new PR as well.